Server/bkr/server/junitxml.py

```diff
@@ -15,13 +15,15 @@
 have their own *different* XML formats.
 """
 
-import urlparse
 import lxml.etree
 from lxml.builder import E
 from bkr.common.helpers import total_seconds
 from bkr.server.util import absolute_url
 from bkr.server.model import TaskStatus, TaskResult
 
+from six.moves import urllib
```

Comment:
I don't think this is needed. Can just delete `urlparse`

Reply:
Correct, I will drop it.